src/components/renderer/OneBlinkFormElements.tsx

```diff
@@ -195,18 +223,10 @@ const FormElementSwitch = React.memo(function OneBlinkFormElement({
   id,
   onChange,
   onLookup,
-}: {
-  formId: number
-  element: FormTypes.FormElement
-  value: unknown | undefined
-  formElementValidation: FormElementValidation | undefined
-  displayValidationMessage: boolean
-  formElementConditionallyShown: FormElementConditionallyShown | undefined
-  id: string
-  isEven: boolean | undefined
-  onChange: FormElementValueChangeHandler
-  onLookup: FormElementLookupHandler
-}) {
+  isDirty,
+  setIsDirty,
```

Comment:
Can just do

```suggestion
 ...dirtyProps,
```
to avoid recreating an object every render.

Reply:
Don't commit that btw, would need to manually change it

Reply:
if other props are added later they'll get mixed up in it, I'll make it a memo

Reply:
If other props get added later, theres work to be done to pass them on anyway, so doesn't matter IMO, but happy to have a memo.